packages/react-integration/demo-app-ts/src/components/demos/SpinnerDemo/SpinnerDemo.tsx

```diff
@@ -10,13 +10,13 @@ export class SpinnerDemo extends React.Component<SpinnerProps> {
   render() {
```

Comment:
You should also add the diameter prop. and an integration test.

Reply:
Done.